cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -4,14 +4,21 @@
 from cfme import test_requirements
 from cfme.intelligence.reports.reports import ImportExportCustomReportsView
 from cfme.intelligence.reports.widgets import ImportExportWidgetsCommitView
-from cfme.utils.path import data_path
+from cfme.utils.fileserver import FileServer
```

Comment:
I don't like that you call this thing a FileServer while I think it really is a some kind of FTP client.

Reply:
I'm really bad at giving a name... I want one FileServer where we can store files and download those while running tests. Yes, its using FTP client. but FileServer I like at that time. please give me some suggestions.

Reply:
On second look I am not sure what this is and why should it exist. It seems that only non-dead-code method in the `FileServer` class is the `__init__` and download. So I am not sure why not to just use the client that there is in the same codebase.

It seems to me this could be named perhaps FTPClientWrapper, but as I told above, I don't know why does it exist on first place.

Reply:
aaaahhh `ftp client` need to send object of the file for `download` and `upload` even some method which I want to restrict. In `ftp.py` you will get the client but the method is more general like we are wrapping python core `ftplib` :). I don't know why we use `send_command` each time. I would like to put that as a client (**As it's more generic**) so created a new file and put restricted implementation you can refer PR header. why we need this and why not `ftp.py`

Reply:
From what I understand , it is using Ftp.py and adding some more methods on top to be used for downloading /uploading files . so yeah FTPClientWrapper or FTPwarapper seems more appropriate :+1:

Reply:
oK... I change complete implementation... @jaryn @sshveta have a look... I try to avoid this but nice to use old utility.